.travis.yml

```diff
@@ -19,14 +15,18 @@ node_js:
     - '5.0'
 
 env:
-    - INFLUX_VERSION=0.9
-    - INFLUX_VERSION=0.10
+    - INFLUX_VERSION=0.9 JUTTLE_PACKAGE=juttle/juttle
+    - INFLUX_VERSION=0.9 JUTTLE_PACKAGE=juttle@0.4.x
+    - INFLUX_VERSION=0.10 JUTTLE_PACKAGE=juttle/juttle
+    - INFLUX_VERSION=0.10 JUTTLE_PACKAGE=juttle@0.4.x
 
 services:
     - docker
 
 before_script:
-    - npm install juttle@0.4.x
+    - rm -rf node_modules/juttle
+    - npm install $JUTTLE_PACKAGE
```

Comment:
The reasons why to test:

a) against latest release juttle - if an influx is released in the between juttle releases, it should still work with existing Juttle stable

b) against the tip of juttle - serves as a red light in the case of breaking changes in master, which is quite frequent.

Or am I missing something?